op-node/rollup/engine/build_invalid.go

```diff
@@ -61,3 +60,12 @@ func (eq *EngDeriver) onBuildInvalid(ev BuildInvalidEvent) {
 	// Signal that we deemed the attributes as unfit
 	eq.emitter.Emit(InvalidPayloadAttributesEvent(ev))
 }
+
+func (eq *EngDeriver) emitDepositsOnlyPayloadAttributesRequest(parentHash common.Hash, derivedFrom eth.L1BlockRef) {
+	eq.log.Warn("Holocene active, requesting deposits-only attributes")
```

Comment:
Let's add the attributes of the request to the log (also see other comment about block-id instead of hash)

Reply:
added the `parent` block id and `derivedFrom`